tests/perf.py

```diff
@@ -37,26 +35,29 @@ def teardown(tmppath):
 
 def setup():
     """Setup test files for performance test."""
+    filename = "testfile"
     tmppath = tempfile.mkdtemp()
-    filepath = join(tmppath, "testfile")
+    filepath = join(tmppath, filename)
 
     # Create test file with random data
     os.system("dd bs=1024 count={1} </dev/urandom >{0}".format(
         filepath, 1024*10))
 
-    return tmppath, filepath
+    return filename, tmppath, filepath
 
 
 #
 # Test methods
 #
-def read_pyfs_chunks(url, chunksize=2097152, n=100):
+def read_pyfs_chunks(url, filename, chunksize=2097152, n=100):
     """Read a file in chunks."""
     t1 = time.time()
 
+    fs = open_fs(url)
+    assert fs.exists(filename)
     i = 0
     while i < n:
-        fsfile = opener.open(url, 'rb-')
+        fsfile = fs.open(filename, 'rb')
```

Comment:
silly question, because it didn't come out when we were looking together at it, does the `-` make a difference ?

Reply:
Thanks for commenting on this. I removed it because it is not supported in `OSFS` in `pyfs` v2: it does not make sense in a normal filesystem. However, I have added back for `xrootd`, which means (see [here](https://github.com/inveniosoftware/xrootdpyfs/blob/master/xrootdpyfs/xrdfile.py#L34)):
`* r-: Open the file for streamed reading; do not allow seek/tell.`